Melanzana.MachO/MachWriter.cs

```diff
@@ -301,6 +302,69 @@ private static void WriteDyldInfoCommand(Stream stream, MachLoadCommandType comm
             stream.Write(dyldInfoHeaderBuffer);
         }
 
+        private static void WriteUuid(Stream stream, MachUuid uuid, bool isLittleEndian)
+        {
+            WriteLoadCommandHeader(
+                stream,
+                MachLoadCommandType.Uuid,
+                LoadCommandHeader.BinarySize + 16,
+                isLittleEndian);
+
+            Span<byte> uuidBuffer = stackalloc byte[16];
+            uuid.Uuid.TryWriteBytes(uuidBuffer);
+            stream.Write(uuidBuffer);
+        }
+
+        private static void WriteSourceVersion(Stream stream, MachSourceVersion sourceVersion, bool isLittleEndian)
+        {
+            WriteLoadCommandHeader(
+                stream,
+                MachLoadCommandType.SourceVersion,
+                LoadCommandHeader.BinarySize + MachSourceVersion.BinarySize,
+                isLittleEndian);
+
+            Span<byte> sourceVersionBuffer = stackalloc byte[MachSourceVersion.BinarySize];
+            sourceVersion.Write(sourceVersionBuffer, isLittleEndian, out var _);
+            stream.Write(sourceVersionBuffer);
+        }
+
+        private static void WriteEncryptionInfo64(Stream stream, MachEncryptionInfo encryptionInfo, bool isLittleEndian)
+        {
+            WriteLoadCommandHeader(
+                stream,
+                MachLoadCommandType.EncryptionInfo64,
+                LoadCommandHeader.BinarySize + MachEncryptionInfo.BinarySize,
+                isLittleEndian);
+
+            Span<byte> encryptionInfoBuffer = stackalloc byte[MachEncryptionInfo.BinarySize];
+            encryptionInfo.Write(encryptionInfoBuffer, isLittleEndian, out var _);
+            stream.Write(encryptionInfoBuffer);
+        }
+
+        private static void WriteRunPath(Stream stream, MachRunPath runPath, bool isLittleEndian, bool is64Bit)
+        {
+            var valueLength = 4 + Encoding.UTF8.GetByteCount(runPath.RunPath) + 1;
```

Comment:
There's couple of null reference type warnings. Is the `RunPath` really supposed to be `string?`?